tests/test_redis_storage.py

```diff
@@ -6,9 +6,9 @@
 
 from redis import RedisError
 
-from thumbor.testing import TestCase
 from tornado.testing import gen_test
 from preggy import expect
+from thumbor.testing import TestCase
```

Comment:
Hey @scorphus, 

Why move this line down?!

Reply:
🤷🏻‍♂️ pylint failed that line. We need isort as part of the lining process then these inconsistencies will stop happening. But that's for another PR.

Reply:
BTW, I'd totally agree with you that it should not be there at all or at least should be part of another commit, that it indeed has nothing to do with Black. But I didn't want to cheat and skip that pre-commit hook (with --no-verify`). Hence, we need isort there too.